near-sdk/tests/code_size.rs

```diff
@@ -21,6 +21,7 @@ fn check_example_size(example: &str) -> usize {
 }
 
 #[test]
+#[ignore = "Size of contracts currently kept in sync with changes"]
```

Comment:
Hmm, isn't the purpose of these tests to check that the contracts do not get too bloated and not that they are out of sync with changes? I am confused how the fact that we enforce that the contracts are in sync with the changes relates to their size being less than 180000 bytes.

Reply:
Because part of our CI checks actually track the exact changes of the size of the contracts, this check is not important anymore. These tests were added before we had the checks in the CI to keep the contracts in line with changes. These tests are also unnecessarily building the contracts which is not ideal to do in the CI